development/CohortOntology.owl

```diff
@@ -781,7 +871,14 @@
     <!-- https://w3id.org/enigma#hasEthnicGroup -->
 
     <owl:DatatypeProperty rdf:about="https://w3id.org/enigma#hasEthnicGroup">
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#Cohort"/>
+        <rdfs:domain>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="https://w3id.org/enigma#Cohort"/>
```

Comment:
Just to clarify, CohortGroups will also have this property because they are cohorts